src/ngraph/op/reshape.cpp

```diff
@@ -143,3 +144,13 @@ void op::Reshape::generate_adjoints(autodiff::Adjoints& adjoints, const NodeVect
 
     adjoints.add_delta(get_argument(0), reshape);
 }
+
+bool op::Reshape::is_dynamic()
```

Comment:
Override shouldn't be necessary here, but we will need it on `DynReshape` and the like.